packages/worker_bee/worker_bee/README.md

```diff
@@ -0,0 +1,131 @@
+# Worker Bee
```

Comment:
We should have a disclaimer stating that this is only intended for usage by amplify-flutter. Otherwise, we will be expected to support this for all manner of use cases. Eventually we can remove this, if the org decides to support it for wider usage.

Reply:
Good thinking. I'll move this to our Quip doc.